examples/stock_example_parallel.jl

```diff
@@ -0,0 +1,97 @@
+#  Copyright 2015, Vincent Leclere, Francois Pacaud and Henri Gerard
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#############################################################################
+# Solving an decision hazard stock problem using multiprocessed dynamic programming :
+# We decide which quantity to store before the randomness happens
+# If too much energy comes, the excess is wasted
+# If not enough energy comes, we lower accordingly what was decided to discharge
+# Min   E [\sum_{t=1}^TF c_t C_t(u_t)]
```

Comment:
I should change the name of C_t. It is a non trivial function of u_t as you can see in the details